src/web/pages/alerts/scpmethodpart.jsx

```diff
@@ -31,6 +31,7 @@ import NewIcon from 'web/components/icon/newicon';
 
 const ScpMethodPart = ({
   prefix,
+  capabilities,
```

Comment:
I think it is better to use `useCapabilities` hook instead of passing it as a prop
```suggestion
```